lib/services/workflow-api-service.js

```diff
@@ -155,7 +237,32 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.find({ node: target })
+            .then(function(workflows){
+                return _.filter(workflows, function(wf) {
+                    return wf.active();
```

Comment:
Why do this work in memory rather than letting the query in `findActiveGraphForTarget` tell the database to do all the processing?